手足.md

```
@@ -0,0 +1,457 @@
+APP_YAML_DIR=./configs/live_specs
+APP_LOG_LEVEL=INFO
+
+# Broker 資格情報は本リポには置かない（別途あなたの実装で）
+# MT5_LOGIN=
+# MT5_PASSWORD=
+# MT5_SERVER=
+
+
+# =============================================================================
+# file: docs/RETRY_POLICY.md  (UseCaseの将来方針メモ：今回は実装しない)
+# =============================================================================
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)